apps/openmw/mwworld/scene.cpp

```diff
@@ -43,22 +43,34 @@ namespace
     }
 
     void updateObjectLocalRotation (const MWWorld::Ptr& ptr, MWPhysics::PhysicsSystem& physics,
-                                    MWRender::RenderingManager& rendering)
+                                    MWRender::RenderingManager& rendering,
+				    int prio = 0)
```

Comment:
`prio` has `bool` semantic, so, I think, it  must be of type `bool`. Also, name `prio` is cryptic. I think its name must reflect multiplication order.